pom.xml

```diff
@@ -23,7 +23,7 @@
     <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
     
     <!-- Fuse version -->
-    <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
```

Comment:
My pleasure... :) - PR updated.  We might as well wait on the Fuse Tooling dependency since that'll change once
   it goes .Final.

    --paull



----- Original Message -----
> From: "Brian Fitzpatrick" <notifications@github.com>
> To: "jboss-switchyard/tools" <tools@noreply.github.com>
> Cc: "Paul Leacu" <pleacu@redhat.com>, "Mention" <mention@noreply.github.com>
> Sent: Tuesday, September 6, 2016 9:47:45 AM
> Subject: Re: [jboss-switchyard/tools] FUSETOOLS-2074 - update 2.1.x for latest bpmn2 modeler (#600)
> 
> > @@ -23,7 +23,7 @@
> >      <tychoExtrasVersion>${tychoVersion}</tychoExtrasVersion>
> >      
> >      <!-- Fuse version -->
> > -
> > <fuse-tooling-url>http://download.jboss.org/jbosstools/mars/development/updates/integration-stack/fuse-tooling/8.0.0.Beta2/all/repo/</fuse-tooling-url>
> 
> @pleacu So the growing consensus with @lhein and @apupier is to split out the
> "fuse-tooling-url" change into a separate JIRA-driven PR. Can we have you
> update this PR to remove that change and put it into a separate PR with a
> new JIRA?
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/jboss-switchyard/tools/pull/600/files/0af89509b68fcf79b0c78eedc1aafc7fb426cf65#r77637122